modules/update-upgrading-oc-adm-upgrade-recommend.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/updating-cluster-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-upgrading-oc-adm-upgrade-recommend_{context}"]
+= Finding recommended update paths with `oc adm upgrade recommend` (Technology Preview)
+
+When updating your cluster, it is useful to find out which versions are recommended for your update. While the `oc adm upgrade` command returns a list of the next available versions, this command decouples that information and narrows down your suggestions to recommend a new target release before you launch your update.
+
+The `oc adm upgrade recommend` command is read-only and will never alter any state in your cluster. 
+
+:FeatureName: The `oc adm upgrade recommend` command
+include::snippets/technology-preview.adoc[]
+
+[IMPORTANT]
+====
+While your cluster does not need to be a Technology Preview-enabled cluster, you must enable the `OC_ENABLE_CMD_UPGRADE_RECOMMEND` Technology Preview environment variable, otherwise the {oc-first} will not recognize the command and you will not be able to use the feature. 
+====
+
+.Procedure 
+
+. Set the `OC_ENABLE_CMD_UPGRADE_RECOMMEND` environment variable to `true` by running the following command:
++
+[source,terminal]
+----
+$ export OC_ENABLE_CMD_UPGRADE_RECOMMEND=true
+----
+. Run one of the following commands:
+
+.. If you want a list of all of the recommended versions for your update, run the `oc adm upgrade recommend` command:
```

Comment:
```suggestion
.. To list the recommended versions for your update, run the `oc adm upgrade recommend` command:
```

Reply:
Occurred to me after the fact that a and b should probably be `**` instead of `..` since IIUC they don't need to be run in order (or even need to both be run at all)